internal/c/libqb/src/qbs__tostr.cpp

```diff
@@ -84,15 +107,18 @@ qbs *qbs__tostr(uint8_t value, int32_t digits, int32_t passed) {
     }
     return tqbs;
 }
+
 // floating points
 qbs *qbs__tostr(float value, int32_t digits, int32_t passed) {
     if (passed) {
         if (digits < 0) {
             error(QB_ERROR_ILLEGAL_FUNCTION_CALL);
             return qbs_new_txt("");
         }
-        if (digits < 1) digits = 1;
-        if (digits > 7) digits = 7;
+        if (digits < 1)
```

Comment:
IMO this is error prone for future extensions, when splitting single line if()s, then braces should be added for both, readability and safeguard in case more instructions are added into the condition later.

Reply:
I agree with you 100%. I used to omit braces for single line `if`s, but I’ve since changed that habit for the same reasons you mentioned. While there is a `clang-format` style that can add braces automatically, it’s not entirely safe to rely on it. Going forward, the best approach is to manually add braces when editing or reviewing code. For now, `clang-format` will simply handle indentation for single line `if`s.